python/semantic_kernel/agents/azure_ai/agent_thread_actions.py

```diff
@@ -130,6 +132,7 @@ async def invoke(
             response_format: The response format.
             parallel_tool_calls: The parallel tool calls.
             metadata: The metadata.
+            polling_options: The polling options defined at the run-level.
```

Comment:
add note that they take precedence over the agent level polling options

Reply:
Added.